landavailability/api/models.py

```diff
@@ -40,6 +42,23 @@ class BusStop(models.Model):
     road = models.CharField(max_length=255, blank=True, null=True)
     nptg_code = models.CharField(max_length=255, blank=True, null=True)
 
+    def update_close_locations(self, distance=1000):
+        locations = Location.objects.filter(
+            geom__dwithin=(self.point, D(m=distance)))
+
+        for location in locations:
+            distance = location.geom.distance(self.point)
+
+            if location.nearest_busstop:
+                if distance > location.geom.distance(
+                        location.nearest_busstop.point):
+                    continue
+
+            location.nearest_busstop = self
+            # FIXME: distance is espressed in a linear way
```

Comment:
I guess the problem here is working out the units in which the results is returned?